core/server/worker/src/main/java/alluxio/worker/block/BlockMetadataManager.java

```diff
@@ -468,4 +484,19 @@ public void resizeTempBlockMeta(TempBlockMeta tempBlockMeta, long newSize)
   public StorageTierAssoc getStorageTierAssoc() {
     return mStorageTierAssoc;
   }
+
+  /**
+   * Get the medium type of the block store location.
+   *
+   * @param location the block store location
+   * @return empty string if block store location is a range, otherwise the medium type
+   */
+  public String getMediumType(BlockStoreLocation location) {
+    if (location.tierAlias().equals(BlockStoreLocation.ANY_TIER)
+        || location.dir() == BlockStoreLocation.ANY_DIR) {
```

Comment:
Is this necessarily true, there could only be one medium type provided in all the tiers or all the dirs in a tier? Do we only need to resolve medium type for specific block store locations?